tests/manager/api/admin/controller/test_report.py

```diff
@@ -43,8 +49,12 @@ def report_fixture(
 
 
 class TestReportController:
+    @patch(
+        "palace.manager.api.admin.controller.report.generate_inventory_and_hold_reports"
```

Comment:
Minor: I've taken to trying to use `patch.object(report, "generate_inventory_and_hold_reports")` since its handled better when using pycharm refactor to do a rename.